cpp/src/search/contains_table.cu

```diff
@@ -134,22 +124,49 @@ std::pair<rmm::device_buffer, bitmask_type const*> build_row_bitmask(table_view
 }
 
 /**
- * @brief Invoke an `operator()` template with a row equality comparator based on the specified
- * `compare_nans` parameter.
+ * @brief Invokes the given `func` with desired comparators based on the specified `compare_nans`
+ parameter.
  *
- * @param compare_nans The flag to specify whether NaNs should be compared equal or not
+ * @tparam HaystackHasNested Flag indicating whether there are nested columns in haystack
+ * @tparam HasAnyNested Flag indicating whether there are nested columns in either haystack or
+ needles
```

Comment:
```suggestion
 * @tparam HasAnyNested Flag indicating whether there are nested columns in either haystack or
          needles
```

Reply:
Not sure it's worth the effort. I assume we shouldn't rely on any manual alignment for code formatting/linting.

Reply:
Regardless of alignment, I think all lines should start with a single space and an asterisk.
https://github.com/rapidsai/cudf/blob/branch-23.10/cpp/doxygen/developer_guide/DOCUMENTATION.md#block-comments
I think this should apply to blank lines within the block comment as well.

Reply:
> all lines should start with a single space and an asterisk.

Oh, good catch. Definitely yes, something went wrong with my IDE formatting. Fixed